input/connect/connect_ps3.c

```diff
@@ -72,17 +72,17 @@ struct __attribute__((__packed__)) sixaxis_activation_report {
 };
 
 union sixaxis_activation_report_f4 {
-   struct sixaxis_activation_report data;
    uint8_t buf[5];
+   struct sixaxis_activation_report data;
 };
 
 union sixaxis_output_report_01 {
-    struct sixaxis_output_report data;
     uint8_t buf[49];
+    struct sixaxis_output_report data;
 };
 
 static const union sixaxis_output_report_01 default_report = {
-    .buf = {
```

Comment:
ISO C90 forbids specifying subobject to initialize

First item of a union will be initialized, which is why the unions were updated above.